src/test/scala/org/allenai/plugins/UtilitiesSpec.scala

```diff
@@ -76,6 +76,19 @@ class UtilitiesSpec extends FlatSpecLike with Matchers with OneInstancePerTest w
     firstHash shouldBe secondHash
   }
 
+  it should "return the same hash when files have the same normalized path" in {
+    // Create versions of fooFile and barFile which are semantically the same, but have different
+    // paths.
+    val fooRelative = new File("foo.txt")
+    val barRelative = new File(tempDirectory, ".." + File.pathSeparatorChar +
+      tempDirectory.getName + File.pathSeparatorChar + "bar.txt")
+
+    val firstHash = Utilities.hashFiles(Seq(fooFile, barFile), tempDirectory)
+    val secondHash = Utilities.hashFiles(Seq(fooFile, barFile), tempDirectory)
```

Comment:
Shouldn't the second hash attempt use a different (denormalized) path to barFile?